src/rgw/driver/rados/rgw_sync.cc

```diff
@@ -1706,6 +1724,12 @@ class RGWMetaSyncShardCR : public RGWCoroutine {
       tn->log(10, "start incremental sync");
       can_adjust_marker = true;
       /* grab lock */
+
+      if (!sync_env->bid_manager->is_highest_bidder(shard_id)) {
```

Comment:
May _is_highest_bidder_(shard_id) may be invoked before spawn RGWMetaSyncShardControlCR ? i mean if rgw could not get the highest bidder, it's not necessary spawn RGWMetaSyncShardControlCR. @smanjara @cbodley

Reply:
`RGWMetaSyncShardControlCR` is where the retry loop lives. we want to run even if we aren't the highest bidder yet